include/api.php

```diff
@@ -560,6 +560,44 @@ function pll_count_posts( $lang, $args = array() ) {
 	return PLL()->model->count_posts( $lang, $args );
 }
 
+/**
+ * Wrapper of `wp_insert_term` with the language.
```

Comment:
```suggestion
 * Wraps `wp_insert_term` with language feature.
```
Sorry I didn't propose earlier.